app/models/draft.rb

```diff
@@ -82,7 +82,7 @@ def self.set(user, key, sequence, data, owner = nil, force_save: false)
         owner: owner
       }
 
-      DB.exec(<<~SQL, opts)
+      draft_id, = DB.query_single(<<~SQL, opts)
```

Comment:
Why the trailing comma?

Reply:
DB.query_single returns an array with a single element. This line means that draft_id will store the first element.

Reply:
Maybe do a first to indicate it's voluntary.

```suggestion
      draft_id = DB.query_single(<<~SQL, opts).first
```